XcodeServerSDK/Server Entities/Repository.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  Repository.swift
+//  XcodeServerSDK
+//
+//  Created by Mateusz Zając on 28.06.2015.
+//  Copyright © 2015 Honza Dvorsky. All rights reserved.
+//
+
+import Foundation
+
+public struct Repository {
```

Comment:
Hmmm... No particular reason but:

- it'll make compiler happy
- this is not inheriting from anything

I can switch to class, no problem - actually implementation stays the same.